src/main/java/org/galatea/starter/utils/helper/HolidayCalculator.java

```diff
@@ -0,0 +1,91 @@
+package org.galatea.starter.utils.helper;
+
```

Comment:
I would recommend not implementing this yourself. Holiday calculations are notoriously finiky. You can skip this.